cli/internal/auth/oauth.go

```diff
@@ -64,7 +74,7 @@ func (a *OauthAuthenticator) Auth(clientValues OauthClientValues) error {
 			AuthURL:  discoveryInfo.AuthorizationEndpoint,
 			TokenURL: discoveryInfo.TokenEndpoint,
 		},
-		RedirectURL: redirectURL,
+		RedirectURL: redirectURLHost + redirectURLPort + redirectURLPath,
```

Comment:
can you please create a small helper function that composes the redirectURL out of the three components and use that one here, as well as in line 131?

Reply:
👍🏽  Added redirectUrlComposer() function to compose urls :)